src/index.html

```diff
@@ -1,14 +1,74 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__link">
+        <a class="logo__img"
+          href="#">
+          <img src="./images/logo.png"
+            alt="Moyo logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="list__items">
+          <li class="list__item">
+            <a href="#"
+              class="item__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#"
```

Comment:
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/fc46bc11-d5c6-45ef-bfdc-9efb47906325)

Reply:
Not fixed